packages/eslint-config-typescript-react/config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  extends: ["@mintlify/eslint-config-typescript", "jsx-a11y", "plugin:import/recommended", "plugin:react/recommended", "plugin:react-hooks/recommended"],
```

Comment:
how do we feel about moving `plugin:import/recommended` to typescript eslint rather than typescript react

Reply:
im down, updated